ui/packages/catalog-ui-search/src/main/webapp/component/search-interactions/search-interactions.hbs

```diff
@@ -11,19 +11,29 @@
  *
  **/
  --}}
-<div class="interaction interaction-type" title="Change the form used to construct the search." data-help="Change the form used to construct the search."></div>
+<div class="interaction interaction-type-text interaction-form" title="Use the Text Search Form to construct the search." data-help="Use the Text Search Form to construct the search.">
+    <div class="interaction-icon fa fa-search"></div>
+    <div class="interaction-text">
+        Text Search
+    </div>
+</div>
+<div class="interaction interaction-type-basic interaction-form" title="Use the Basic Search Form to construct the search." data-help="Use the Basic Search Form to construct the search.">
+    <div class="interaction-icon fa fa-search"></div>
+    <div class="interaction-text">
+        Basic Search
+    </div>
+</div>
 <div class="interaction interaction-type-advanced interaction-form" title="Use the Advanced Search Form to construct the search." data-help="Use the Advanced Search Form to construct the search.">
     <div class="interaction-icon fa fa-search"></div>
     <div class="interaction-text">
         Advanced Search
     </div>
 </div>
-
+<div class="interaction interaction-type" title="Change the form used to construct the search." data-help="Change the form used to construct the search."></div>
```

Comment:
✏️ 	Could make this a self closing `<div/>`

Reply:
A self-closing div breaks marionette for some reason. I get `an el undefined must exist in dom` error 🤷‍♂️